### PR TITLE
added normalize

### PR DESCRIPTION
**Pull request by hermela11**

### CODE REVIEW

File: scripts/scil_evaluate_connectivity_pairwise_agreement_measures.py [hunk 1]
```diff
@@ -35,6 +35,8 @@ def _build_arg_parser():
                    help='Path of the output json file.')
     p.add_argument('--single_compare',
                    help='Compare inputs to this single file.')
+    p.add_argument('--normalize', action='store_true',
+                   help='If set, will normalize all matricies zero to one.')
```
Comment (frheault) [inline scripts/scil_evaluate_connectivity_pairwise_agreement_measures.py:39]: Not zeros, the maximum values will be one
*matrices

**Review by frheault — APPROVED** [review]
After that small fix, ready to go

File: scripts/scil_evaluate_connectivity_pairwise_agreement_measures.py (hunk 1 above)
Comment (hermela11) [reply scripts/scil_evaluate_connectivity_pairwise_agreement_measures.py:39]: I meant the range of 0-1. Should I reword to add that in?

Comment (frheault) [reply scripts/scil_evaluate_connectivity_pairwise_agreement_measures.py:39]: Ahhhh! I understand.
Maybe '**from** zero to one.' is enough then.

But right now you are not changing the smaller value. Can you add something for that
something like `tmp_mat -= np.min(tmp_mat)`

**Pull request merged**